src/WhereConstraints/WhereConstraintsDirective.php

```diff
@@ -225,4 +223,21 @@ function (string $columnName): string {
 
         return PartialParser::enumTypeDefinition($enumDefinition);
     }
+
+    /**
+     * Ensure the column name is well formed and prevent SQL injection.
+     *
+     * @param  string  $column
+     * @return void
+     *
+     * @throws \GraphQL\Error\Error
+     */
+    protected static function assertValidColumnName(string $column): void
+    {
+        if (! \Safe\preg_match('/^(?![0-9])[A-Za-z0-9_-]*$/', $column)) {
```

Comment:
This prevents using columns that start with a number or include a space, both of which I believe many databases allow. They're funky to work with in Laravel (eg. `$model->{'1 fun column'}`) but are possible.

Reply:
I don't want to risk SQL injection. There is always the option to define the columns explicitely, that way any name can be used.